lib/workers/repository/update/pr/body/changelogs.ts

```diff
@@ -11,23 +11,11 @@ export function getChangelogs(config: BranchConfig): string {
     return releaseNotes;
   }
 
-  const countReleaseNodesByRepoName: Record<string, number> = {};
-
-  for (const upgrade of config.upgrades) {
-    if (upgrade.hasReleaseNotes && upgrade.repoName) {
-      countReleaseNodesByRepoName[upgrade.repoName] =
-        (countReleaseNodesByRepoName[upgrade.repoName] || 0) + 1;
-    }
-  }
-
   for (const upgrade of config.upgrades) {
     if (upgrade.hasReleaseNotes && upgrade.repoName) {
-      // TODO: types (#7154)
-      upgrade.releaseNotesSummaryTitle = `${upgrade.repoName}${
-        countReleaseNodesByRepoName[upgrade.repoName] > 1
-          ? ` (${upgrade.depName!})`
-          : ''
-      }`;
+      upgrade.releaseNotesSummaryTitle = `${
+        upgrade.repoName
+      } (${upgrade.depName!})`;
```

Comment:
@rarkins won't this duplicate release notes again for each package of a mono repo? as again all titles are different? if yes this PR makes it worse than better.

Reply:
I haven't tested this with a monorepo (with multiple packages), but as far as I currently understand this scenario doesn't change. Monorepo's already had the brackets after a component name, now just every repo has this

Reply:
i hope you are right, can you please create a test repo for that?